bylaws/009-per-workflow.md

```diff
@@ -0,0 +1,93 @@
+
+Once four weeks have passed and two viable trial implementations can be demonstrated, the Sponsor may call an Acceptance Vote. If the Acceptance Vote fails, the specification may remain in Review.
+
+## Accepted
+
+If the Acceptance Vote passes, then the proposal officially becomes an accepted PER.  It is considered the 1.0.0 release of that PER.
+
+## Evolution
+
+Unlike a PSR, a PER is expected to evolve over time.  Once the initial version of a PER is approved, the Working Group remains active but its minimum requirements drop to an Editor and two additional individuals.  By default, the Working Group at time of first approval continues in that role.  The Working Group may continue to develop the PER over time as the context of the recommendation evolves, such as newly released PHP language versions, additional input from third parties, etc.
+
+For any proposed change to the PER, the Editor must notify the Core Committee via a post to the mailing list of an Intent to Merge a given change.
+
+If no member of the Core Committee objects within seven days, the proposed change is assumed accepted by default.  Any member of the Core Committee may request during that seven-day period that the change be put to a vote.  If any member does so, the change is immediately put to an Acceptance Vote to determine if it is accepted or not.
+
+The Editor is empowered to declare tagged versions of a PER at any time.  Tagged versions MUST follow Semantic Versioning conventions.  However, any change that would necessitate a new Major release MUST be put to an Acceptance Vote by the Core Committee.
+
+## Staffing Changes
+
+The Editor of an approved PER is empowered to add or remove members from the Working Group at any time, with notification to the Core Committee.
+
+Should the Editor of a PER Working Group resign or become inactive, the Core Committee may appoint a new Editor.  Preference will be given to the existing Working Group members at that time.
+
+Should a Working Group become inactive for 
+
+## Deprecated
+
+A Deprecated PSR is one that has been approved, but is no longer considered relevant or recommended. Typically this is due to the PSR being superseded by a new version, but that is not required.
+
+A PSR may be Deprecated explicitly as part of the Acceptance Vote for another PSR. Alternatively, it may be marked Deprecated by a Deprecation Vote.
+
+## Abandoned
+
+A PER Working Group can be marked as Abandoned by Secretaries when it is without an Editor for 60 days. After a period of 6 months without significant activity in a Working Group, the Secretaries may also declare a PER Working Group to be Abandoned.
+
+Should in the future a new Editor wish to establish a new Working Group to continue development of the same PER, the candidate may petition the Core Committee to do so by asking for a new Entrance Vote.  Only the Editor and two additional individuals are required at this time.
+
+## PER Working Group Fast-track
```

Comment:
I don't like the name for this section.  Please suggest something better.

Reply:
```suggestion
## Expedited PER Working Group
```